src/org/connection.ts

```diff
@@ -231,10 +217,17 @@ export class Connection<S extends Schema = Schema> extends JSForceConnection<S>
         contentType: 'application/json',
       });
 
-      const url = `${this.baseUrl()}/metadata/deployRequest`;
-
-      const httpRequest: HttpRequest = { method: 'POST', url, headers, body: form };
-      return this.request(httpRequest);
+      const request: HttpRequest = {
+        url: '/metadata/deployRequest',
```

Comment:
`jsforce` resolves the url to include `baseUrl` and any other things correct?

Reply:
yes, see jsdoc on https://github.com/jsforce/jsforce/blob/7d254241799b8433572cbb435ef4127d2add90b9/src/connection.ts#L645